docs/Data_Portal/Users_Guide/oncomatrix.md

```diff
@@ -353,6 +449,7 @@ To reset the zoom level to default, click on the `Reset` button as shown. This w
 
 [![Reset Zoom](./images/oncomatrix/55-reset_zoom.png)](./images/oncomatrix/55-reset_zoom.png 'Click to see the full image.')
```

Comment:
@siosonel ,  Here's an example of an image that should render correctly.

Reply:
Hi Bill, We'll switch to using full markdown and edit the image file size itself using a photo editor, instead of via html or markdown. Please give us maybe 20 minutes to commit this change.